Modix/Modules/CatModule.cs

```diff
@@ -0,0 +1,178 @@
+﻿namespace Modix.Modules
```

Comment:
The code to actually query for, and download the cat image would be better placed in a service class.